### ISSUE TITLE
New triggers do not allow you to choose the correct variable.

### ISSUE THREAD

**Issue by WeylonSantana** [AscensionGameDev/Intersect-Engine#659]
**Describe the bug**
I downloaded version 114, and updated a version I had of 109, a clean version where I do tests, created a variable and went to test the new condition to know what I can do, when I choose the variable, and save the event, when I open the event the variable is another.

**Version Encountered**
0.7.0.114

**To Reproduce**
1. Create a new var
2. Create a new common event
3. Change trigger to var change and choose the new var created, save and reopen event

**Expected behavior**
keep the variable I chose

**Screenshots**

**Logs**
If there are error logs please attach them below.

**Operating System:**
Windows

**Additional context**
Basically this is the same error from here(https://github.com/AscensionGameDev/Intersect-Engine/issues/511), but in the event trigger
